.gitmodules

```diff
@@ -16,4 +16,4 @@
         branch = mpich/main
 [submodule "modules/yaksa"]
 	path = modules/yaksa
-	url = https://github.com/pmodels/yaksa
+	url = https://github.com/wesbland/libraries.runtimes.hpc.yaksa-aurora.git
```

Comment:
Separate this commit -- I understand this is temporary?

Reply:
Well, the particular commit that this submodule update is point to is temporary, though I'd argue that this is where the final update should go. It'll just need to change to pointing to the main repo instead of my fork.